test/functional/test_site/expected/bugs/index.html

```diff
@@ -12,7 +12,8 @@
     <link rel="stylesheet" href="../markbind/css/bootstrap-glyphicons.min.css" >
     <link rel="stylesheet" href="../markbind/css/github.min.css">
     <link rel="stylesheet" href="../markbind/css/markbind.css">
-    <link rel="stylesheet" href="../markbind/layouts/default/styles.css">
+            <link rel="stylesheet" href="STYLESHEET_LINK">
+            <link rel="stylesheet" href="../markbind/layouts/default/styles.css">
```

Comment:
Indentation is off.

Reply:
Sorry, accidentally auto indented that part in `page.ejs` 😅